src/screens/HomeScreen.js

```diff
@@ -106,6 +108,8 @@ export const HomeScreen = ({ navigation }) => {
     })
   };
 
+  useMatomoTrackScreenView(MATOMO_TRACKING.SCREEN_VIEW.HOME);
```

Comment:
could be moved up as well

Reply:
done in https://github.com/ikuseiGmbH/smart-village-app-app/pull/109/commits/f22a1566b20d2eded43188bff5bc48e012167c41